launch/vn200.launch

```diff
@@ -2,15 +2,18 @@
 
 	<node pkg="vectornav" type="vn200_node" name="vectornav"  output="screen" > 
 	  
-	  <param name="serial_port" type="string" value="/dev/ttyUSB0" />
+	  <param name="serial_port" type="string" value="/dev/ttyTHS0" />
```

Comment:
This seems like a less useful default for the average user. But I might not care. :)